src/extensions/react-flow/nodes/NodeContent.tsx

```diff
@@ -341,11 +349,16 @@ export function NodeContent<CONTENT_PROPS = any>({
 
     const { handles = defaultHandles(flowVersionCheck), ...otherProps } = otherDomProps;
 
-    const isResizeable = !!onNodeResize && minimalShape === "none";
+    const isResizable =
+        typeof onNodeResize === "function" && Object.keys(resizeDirections).length && minimalShape === "none";
```

Comment:
This conditional check would activate resizing also for e.g. `resizeDirections={bottom: false}`, what would be not correct. At least one value for `bottom`, `right` and `bottomRight` need to be `true`